x/margin/types/genesis.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+func DefaultGenesis() *GenesisState {
+	return &GenesisState{
+		// TODO review default param values
+		Params: &Params{
+			LeverageMax: sdk.NewUint(0),
+		},
```

Comment:
@AustinoBombino are there default values we should be using here for the params?

Reply:
Yes, this will be something we'll have to tune. But we can start it at 1.